iceberg/iceberg-handler/src/test/results/positive/llap/iceberg_merge_schema.q.out

```diff
@@ -0,0 +1,163 @@
+PREHOOK: query: CREATE TABLE calls (
```

Comment:
Please move this q.out file to `iceberg/iceberg-handler/src/test/results/positive/` and that should fix the only failed ptest in the last run. It should be run by `TestIcebergCliDriver`.